docker/tests/sidecar/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# OpenJDK distributed under GPLv2+Oracle Classpath Exception license (http://openjdk.java.net/legal/gplv2+ce.html)
+# Alpine Linux packages distributed under various licenses including GPL-3.0+ (https://pkgs.alpinelinux.org/packages)
+FROM dotcms/dotcms-seed as build-cms
+
+LABEL com.dotcms.contact "support@dotcms.com"
+LABEL com.dotcms.vendor "dotCMS LLC"
+LABEL com.dotcms.description "dotCMS Content Management System"
+
+#  One of ["TARBALL_URL", "RELEASE", "NIGHTLY", "COMMIT", "TAG" ]
+ARG BUILD_FROM=COMMIT
+
+#  Value resolved in the context of $BUILD_FROM
+ARG BUILD_ID=HEAD
+
+WORKDIR /srv
+
+# dotCMS core distributed under GPLv3 license (https://github.com/dotCMS/core/blob/master/license.txt)
+COPY build-src/ /build/
+RUN chmod 500 /build/build_dotcms.sh && /build/build_dotcms.sh ${BUILD_FROM} ${BUILD_ID}
+
+RUN mkdir -p /srv/utils /srv/templates /srv/config /srv/home
+RUN chmod -R 666 /srv && find /srv/ -type d -exec chmod a+x {} \;
+
+##
+## Begin target image build
+##
+
+# OpenJDK distributed under GPLv2+Oracle Classpath Exception license (http://openjdk.java.net/legal/gplv2+ce.html)
+# Alpine Linux packages distributed under various licenses including GPL-3.0+ (https://pkgs.alpinelinux.org/packages)
+FROM openjdk:8-jdk-alpine
+
+WORKDIR /srv
+
+RUN apk --no-cache upgrade \
+    && apk add --no-cache bash openssl ca-certificates gnupg grep sed tini nss s6-dns \
+    && update-ca-certificates
+
+RUN apk --no-cache add libc6-compat tomcat-native bash grep curl && apk add libsass --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/community/
+
+# This is a nasty hack for libjsass, Alpine 3.9 no longer has a deprecated symlink in /lib
+RUN [ ! -e /lib/ld-linux-x86-64.so.2 ] &&  ln -s $(readlink /lib64/ld-linux-x86-64.so.2) /lib/ld-linux-x86-64.so.2 || :
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Quote this to prevent word splitting.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5442148)